website/_enemies/generic/key.md

```diff
@@ -7,4 +7,6 @@ hp: 1
 sprite_folder_name: key
 ---
 
-When killed, will fly to its associated [Door](#enemy-door-for-key) and ~~unlock~~ kill it.
+When killed, will fly to its associated [Door](#enemy-door-for-key) and kill it.
```

Comment:
Killing a door sounds weird even though it is technically correct in the game. I think it needs better wording.